src/components/SignUp/SignUpMETA.js

```diff
@@ -128,21 +128,32 @@ export const CareerData = {
     'Other',
   ],
 };
+
 export const Options = [
     'Study abroad',
-    'Get a new job (e.g., Interview, job search..) ',
+    'Career advice (e.g., interview, job search..)',
+    'Portfolio & Resume',
+    'Startup',
+    'Career change',
+    'Networking',
+    'Soft skills (e.g., communication..)',
+];
+
+export const OptionsFilter = [
+    'Study abroad',
+    'Career advice',
     'Portfolio & Resume',
-    'Start up',
+    'Startup',
     'Career change',
     'Networking',
-    'Soft skills (e.g., Communication..)',
+    'Soft skills',
```

Comment:
In filter, we decide delete after e.g. , however, I send data with original string.